diff --git a/Externals/stb/deprecated/stb.h b/Externals/stb/deprecated/stb.h
index 1633c3b..6cda830 100644
--- a/Externals/stb/deprecated/stb.h
+++ b/Externals/stb/deprecated/stb.h	
@@ -5609,7 +5609,7 @@ static FILE *stb__open_temp_file(char *temp_name, char *src_name, const char *mo
    #else
    {
       stb_p_strcpy_s(temp_name+p, 65536, "stmpXXXXXX");
-      #ifdef __MINGW32__
+      #if 0
          int fd = open(stb_p_mktemp(temp_name), O_RDWR);
       #else
          int fd = mkstemp(temp_name);
@@ -6138,7 +6138,7 @@ static char **readdir_raw(char *dir, int return_subdirs, char *mask)
    char buffer[4096], with_slash[4096];
    size_t n;
 
-   #ifdef WIN32
+   #ifdef _WIN32
       stb__wchar *ws;
       struct _wfinddata_t data;
    #ifdef _WIN64
@@ -6165,7 +6165,7 @@ static char **readdir_raw(char *dir, int return_subdirs, char *mask)
    if (!stb_strscpy(with_slash,buffer,sizeof(with_slash)))
       return NULL;
 
-   #ifdef WIN32
+   #ifdef _WIN32
       if (!stb_strscpy(buffer+n,"*.*",sizeof(buffer)-n))
          return NULL;
       ws = stb__from_utf8(buffer);
@@ -6176,7 +6176,7 @@ static char **readdir_raw(char *dir, int return_subdirs, char *mask)
 
    if (z != none) {
       int nonempty = STB_TRUE;
-      #ifndef WIN32
+      #ifndef _WIN32
       struct dirent *data = readdir(z);
       nonempty = (data != NULL);
       #endif
@@ -6185,7 +6185,7 @@ static char **readdir_raw(char *dir, int return_subdirs, char *mask)
 
          do {
             int is_subdir;
-            #ifdef WIN32
+            #ifdef _WIN32
             char *name = stb__to_utf8((stb__wchar *)data.name);
             if (name == NULL) {
                fprintf(stderr, "%s to convert '%S' to %s!\n", "Unable", data.name, "utf8");
@@ -6213,13 +6213,13 @@ static char **readdir_raw(char *dir, int return_subdirs, char *mask)
                }
             }
          }
-         #ifdef WIN32
+         #ifdef _WIN32
          while (0 == _wfindnext(z, &data));
          #else
          while ((data = readdir(z)) != NULL);
          #endif
       }
-      #ifdef WIN32
+      #ifdef _WIN32
          _findclose(z);
       #else
          closedir(z);
@@ -12342,7 +12342,7 @@ static void * stb__io_task(void *p)
       return 0;
    }
    if (dc->f) {
-      #ifdef WIN32
+      #ifdef _WIN32
       f = _fdopen(_dup(_fileno(dc->f)), "rb");
       #else
       f = fdopen(dup(fileno(dc->f)), "rb");
